### DISCUSSION TITLE
Support listener mode for SRP re-publishing

### DISCUSSION THREAD

**Discussion by Xeue** [OvenMediaLabs/OvenMediaEngine#1341] [Ideas]
Hi guys, you've done some great work with OME, I'm missing one piece to round of our workflows.
Often we have both encoders and decoders at locations where we aren't able to do port forwarding/firewall navigation. In these scenarios the server being in listener mode is extremely useful. I would image this would behave similarly to SRT Inputs, add a section in <bind> and then add am entry in <publishers> along the lines of <SRTListener>.
I havn't had the time to dig into the code to see how much pain this is to implement, but if I ever get the chance/time I will try and would aim to make my own PR for it.

**Comment by Ogog135** [comment]
Ola!4

**Reply by krisdahl** [reply]
You’re basically asking for an Output Publisher that is an SRT listener?  I would love to have that too.

**Reply by Xeue** [reply]
Yes exactly, there's lot of workflows where having your output as a listener is advantageous.

**Comment by getroot** [comment]
This is in our backlog. But it's not a high priority.

We have the principle that the client function (SRT Push) can utilize another open source project such as ffmpeg, but the server function (SRT Publisher) is implemented independently. Therefore, this will take quite some time to implement.
